### PR TITLE
Update kubeadm_init.md, add `kubeadm_init` execute phases

### PR DESCRIPTION
**Pull request by mavk**
add `kubeadm_init` execute phases

### CODE REVIEW

File: content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md [hunk 1]
```diff
@@ -58,6 +59,47 @@ addon                        Install required addons for passing conformance tes
...
 show-join-command            Show the join command for control-plane and worker node
 ```
+-->
+```
+preflight                    预检
+certs                        生成证书
+  /ca                          生成自签名根 CA 用于配置其他 kubernetes 组件
+  /apiserver                   生成 apiserver 的证书
+  /apiserver-kubelet-client    生成 apiserver 连接到 kubelet 的证书
+  /front-proxy-ca              生成前端代理自签名CA(扩展apiserver)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md:69]:
```suggestion
  /front-proxy-ca              生成前端代理自签名 CA（扩展 apiserver）
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md [hunk 2]
```diff
@@ -58,6 +59,47 @@ addon                        Install required addons for passing conformance tes
...
+  /front-proxy-client          生成前端代理客户端的证书(扩展apiserver)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md:70]:
```suggestion
  /front-proxy-client          生成前端代理客户端的证书（扩展 apiserver）
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md [hunk 3]
```diff
@@ -58,6 +59,47 @@ addon                        Install required addons for passing conformance tes
...
+  /etcd-ca                     生成 etcd 自签名 CA
+  /etcd-server                 生成 etcd 服务的证书
```
Comment (tengqm) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md:72]:
```suggestion
  /etcd-server                 生成 etcd 服务器证书
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md [hunk 4]
```diff
@@ -58,6 +59,47 @@ addon                        Install required addons for passing conformance tes
...
+  /etcd-peer                   生成 etcd 节点相互通信的证书
+  /etcd-healthcheck-client     生成 etcd 健康检查的证书
+  /apiserver-etcd-client       生成 apiserver 访问 etcd 的证书
+  /sa                          生成用于签署服务帐户 token 的私钥和公钥
```
Comment (tengqm) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md:76]:
```suggestion
  /sa                          生成用于签署服务帐户令牌的私钥和公钥
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md [hunk 5]
```diff
@@ -58,6 +59,47 @@ addon                        Install required addons for passing conformance tes
...
+  /kubelet                     为 kubelet 生成一个 kubeconfig 文件，*仅*用于集群引导
+  /controller-manager          生成 kubeconfig 文件供控制器管理器使用
+  /scheduler                   生成 kubeconfig 文件供调度程序使用
+etcd                         为本地 etcd 生成静态 Pod 清单文件
+  /local                       为本地单节点本地 etcd 实例生成静态 Pod 清单文件
+control-plane                生成建立控制平面所需的所有静态 Pod 清单文件
+  /apiserver                   生成 kube-apiserver 静态 Pod 清单
+  /controller-manager          生成 kube-controller-manager 静态 Pod 清单
+  /scheduler                   生成 kube-scheduler 静态 Pod 清单
+kubelet-start                写入 kubelet 设置并 启动/重启 kubelet
```
Comment (tengqm) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md:89]:
```suggestion
kubelet-start                写入 kubelet 设置并启动（或重启） kubelet
```

**Comment by windsonsea** [comment]
ping @mavk , would you have time to resolve comments and perform rebase?

**Comment by mavk** [comment]
@windsonsea ofcous, wait for me.

**Pull request closed without merging**